src/librustdoc/json/conversions.rs

```diff
@@ -0,0 +1,597 @@
+use std::convert::From;
+
+use rustc_ast::ast;
+use rustc_span::def_id;
+
+use crate::clean;
+use crate::doctree;
+use crate::formats::item_type::ItemType;
+use crate::json::types::*;
+
+impl From<clean::Item> for Item {
```

Comment:
Although there may be little point in adding docs on each of the items in this module, a top-level module docstring "//!" could be helpful.